cookbooks/clearwater/recipes/vellum.rb

```diff
@@ -32,11 +32,7 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-package "vellum-node" do
-  action [:install]
-  options "--force-yes"
-end
-package "homer-cassandra" do
```

Comment:
Probably, but I don't see any harm in including memento-cassandra on Vellum here, and I wasn't sure if a deployment resize would pick up the addition of memento to the vellum nodes, so I figured it was safer to leave in.